### PR TITLE
make test-cmd more reliable on travis by pre-minting certs

### PR DESCRIPTION
**Pull request by deads2k**
I think test-cmd.sh is often failing on travis because of the time it takes to mint all the certs. I'm pre-minting the certs to make the time it takes to complete `openshift start` smaller.  I also prevented test-cmd.sh from self-destructing on non-virgin systems.

@liggitt

### CODE REVIEW

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -70,6 +72,18 @@ echo openshift: $out
...
+echo "[INFO] Create certificates for the OpenShift server"
+# find the same IP that openshift start will bind to.  This allows access from pods that have to talk back to master
+ALL_IP_ADDRESSES=`ifconfig | grep "inet " | awk '{print $2}'`
+SERVER_HOSTNAME_LIST="${PUBLIC_MASTER_HOST},localhost"
+while read -r IP_ADDRESS
+do
+    SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},${IP_ADDRESS}"
+done <<< "${ALL_IP_ADDRESSES}"
+
+openshift admin create-all-certs --overwrite=false --cert-dir="${CERT_DIR}" --hostnames="${SERVER_HOSTNAME_LIST}" --nodes="127.0.0.1" --master="${MASTER_ADDR}" --public-master="${API_SCHEME}://${PUBLIC_MASTER_HOST}"
```
Comment (liggitt) [inline hack/test-cmd.sh]: `start` is using API_HOST as the nodename (`--hostname="${API_HOST}"`)... should match that in create-all-certs

**Comment by liggitt** [comment]
LGTM otherwise

**Comment by deads2k** [comment]
[merge]

**Comment by deads2k** [comment]
[merge]

**Pull request merged**